docker/README.md

```diff
@@ -163,4 +171,4 @@ Once the updated `nodes.json` file is committed to master, the new images can be
 1. In the Network Designer sidebar, click on the **Show All Versions** toggle
 1. At the bottom of the node list, click on the **Check for new Node Versions** link
 1. A dialog will open displaying the new versions available
-1. Click the **Add New Versions** button to begin using them
+1. Click the **Add New Versions** button to begin using them
```

Comment:
I think this just removed the last newline char from this line. Can you add it back to remove this diff?